package.json

```diff
@@ -36,12 +36,12 @@
     "access": "public"
   },
   "engines": {
-    "node": ">=16.0.0"
+    "node": ">=18.0.0"
```

Comment:
I'm not sure I really thought this through when I updated this value in the Node Core. _Technically_, this is a breaking change. I didn't necessarily want existing SDKs to have to release new major versions just to use the major version of the core.

Is it possible to install the new version of the core without making this change?

Reply:
discussed offline